docker-compose.yaml

```diff
@@ -146,15 +143,15 @@ services:
     image: httpd:2.4
     volumes:
       - ${AIRFLOW_PROJ_DIR:-.}/weather:/usr/local/apache2/htdocs
-  
+
   postgres:
     image: postgres:13
     environment:
       POSTGRES_USER: airflow
       POSTGRES_PASSWORD: airflow
       POSTGRES_DB: airflow
     volumes:
-      - postgres-db-volume:/var/lib/postgresql/data
+      - ${AIRFLOW_PROJ_DIR:-.}/db:/var/lib/postgresql/data
```

Comment:
this is an important change! as far as i can tell the current `atd-data03` does not have a persisted postgres database.